Features/Stories/FieldTypes/IntegerFieldType.feature

```diff
@@ -0,0 +1,202 @@
+Feature: Test the validations done on fields from PlatformUI - Integer fieldtype
+    In order to validate the integer fieldtype
+    As an Editor  user
+    I need to be able to create, update and delete content with integer fieldtypes
```

Comment:
deletion is not covered here, and neither should it as long as there is no logic around integer that needs to be verified on delete.